### PR TITLE
Add example and unit test for passing a comma separated list to -f option

### PR DESCRIPTION
**Pull request by mgianatagh**
Add example and unit test for passing a comma separated list to -f option

### CODE REVIEW

File: tools/vz/cmd/install/install.go [hunk 1]
```diff
@@ -38,6 +38,8 @@ vz install --version v1.3.0 --set profile=dev --timeout 20m
 vz install --version v1.3.0 --set profile=dev --set components.kiali.enabled=false
 
 # Install the latest version of Verrazzano using CR overlays and explicit value sets.  Output the logs in json format.
+# The overlay files can be a comma separated list or a series of -f options.  Both formats are shown.
+vz install -f base.yaml,custom.yaml --set profile=prod --log-format json
```
Comment (rosemarymarano) [inline tools/vz/cmd/install/install.go:42]: base.yaml,custom.yaml -> no space here correct?
Also, is it correct to refer to json lowercase? Usually it's JSON.

Comment (mgianatagh) [reply tools/vz/cmd/install/install.go:42]: correct, no spaces in the comma-separated list

In this case the lowercase reference to json is correct

**Review by rosemarymarano — COMMENTED** [review]
So, just the change to "comma-separated" and I approve.

File: tools/vz/cmd/install/install.go [hunk 2]
```diff
@@ -38,6 +38,8 @@ vz install --version v1.3.0 --set profile=dev --timeout 20m
 vz install --version v1.3.0 --set profile=dev --set components.kiali.enabled=false
 
 # Install the latest version of Verrazzano using CR overlays and explicit value sets.  Output the logs in json format.
+# The overlay files can be a comma separated list or a series of -f options.  Both formats are shown.
```
[the remark this answers is not in the document]
Comment (mgianatagh) [reply tools/vz/cmd/install/install.go]: fixed

**Review by agv426 — APPROVED** [review]

**Pull request merged**